src/client/java/teammates/client/scripts/DataMigrationForFeedbackQuestionsToQuestions.java

```diff
@@ -0,0 +1,137 @@
+import teammates.common.exception.InvalidParametersException;
+import teammates.common.util.Assumption;
+import teammates.logic.api.Logic;
+import teammates.storage.api.FeedbackQuestionsDb;
+import teammates.storage.api.QuestionsDb;
+import teammates.storage.datastore.Datastore;
+import teammates.storage.entity.FeedbackQuestion;
+
+/**
+ * Script to create a Question copy of old FeedbackQuestions.
+ * 
+ */
+public class DataMigrationForFeedbackQuestionsToQuestions extends RemoteApiClient {
+    
+    private final boolean isPreview = true;
+    
+    private enum ScriptTarget {
+        BY_TIME, BY_COURSE, ALL;
+    }
+    
+    ScriptTarget target = ScriptTarget.BY_TIME;
+    
+    private final int numDays = 100;
+    private final String courseId = "";
+    
+    
+    public static void main(String[] args) throws IOException {
+        new DataMigrationForFeedbackQuestionsToQuestions().doOperationRemotely();
+    }
+
+    @Override
+    protected void doOperation() {
+        
+        Datastore.initialize();
+        List<FeedbackQuestion> feedbackQuestions;
+        if (target == ScriptTarget.BY_TIME) {
+            Calendar startCal = Calendar.getInstance();
+            startCal.add(Calendar.DAY_OF_YEAR, -1 * numDays);
+            
+            feedbackQuestions = getOldQuestionsSince(startCal.getTime());
+            
+        } else if (target == ScriptTarget.BY_COURSE) {
+            feedbackQuestions = getFeedbackQuestionEntitiesForCourse(courseId);
+            
+        } else if (target == ScriptTarget.ALL) {
+            feedbackQuestions = getAllOldQuestions();
+            
+        } else {
+            feedbackQuestions = null;
+            Assumption.fail("no target selected");
+        }
+        
+        List<FeedbackQuestionAttributes> feedbackQuestionAttributes =
+                FeedbackQuestionsDb.getListOfQuestionAttributes(feedbackQuestions);
+        for (FeedbackQuestionAttributes old : feedbackQuestionAttributes) {
+            FeedbackSessionAttributes session = new Logic().getFeedbackSession(old.getFeedbackSessionName(), old.getCourseId());
+            if (session == null) {
+                System.out.println("question: " + old.getIdentificationString());
+                System.out.println(String.format("error finding session %s",
+                                                 old.getFeedbackSessionName() + ":"
+                                                 + old.getCourseId()));
+                System.out.println("possibly due to orphaned responses");
+                continue;
+            }
+            
+            if (isPreview) {
+                FeedbackQuestionAttributes existingQn =
+                        new QuestionsDb().getFeedbackQuestion(old.feedbackSessionName, old.courseId, old.getId());
+                if (existingQn == null) {
+                    System.out.println("Will create question: " + old.getIdentificationString());
```

Comment:
For consistency, maybe this could start with a lower case `w`, or perhaps the other print statements could begin with an upper case letter